catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -57,8 +57,9 @@ const SearchLayout: FunctionComponent<{ hasEventsExhibitions: boolean }> = ({
 
   const defaultPageLayoutMetadata: PageLayoutMetadata = {
     ...basePageMetadata,
-    title: 'Search Page',
-    description: 'TBC',
+    title: 'Search | Wellcome Collection',
```

Comment:
You don't need to append `| Wellcome Collection` here; it's added on all pages by the PageLayout component: https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/PageLayout/PageLayout.tsx#L84-L87

It's getting doubled if I run this locally:

<img width="1005" alt="Screenshot 2023-01-10 at 14 11 01" src="https://user-images.githubusercontent.com/301220/211573510-dab49f11-f06c-4d85-977c-42bf82917efe.png">